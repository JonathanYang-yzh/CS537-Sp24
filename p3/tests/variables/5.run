bash -c "$myenvvar"
export myenvvar=uname
bash -c "$myenvvar"
export myenvvar=
bash -c "$myenvvar"