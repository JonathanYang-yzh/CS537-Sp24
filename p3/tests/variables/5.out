${PROMPT}bash: line 1: : command not found
${PROMPT}${PROMPT}Linux
${PROMPT}${PROMPT}bash: line 1: : command not found
${PROMPT}